src/UIComponents/ResultsTable.jsx

```diff
@@ -0,0 +1,106 @@
+/* React component to handle displaying test data and A.I. Bot's guesses. */
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import {
+  getConvertedAccuracyCheckExamples,
+  getConvertedLabels
+} from "../redux";
+import { styles, colors } from "../constants";
+
+class ResultsTable extends Component {
+  static propTypes = {
+    selectedFeatures: PropTypes.array,
+    labelColumn: PropTypes.string,
+    accuracyCheckExamples: PropTypes.array,
+    accuracyCheckLabels: PropTypes.array,
+    accuracyCheckPredictedLabels: PropTypes.array
+  };
+
+  render() {
+    return (
+      <div>
+        <div style={styles.floatLeft}>
+          <span style={styles.largeText}>Features</span>
+          <table>
+            <thead>
+              <tr style={{ backgroundColor: colors.feature }}>
```

Comment:
We could also have something in `styles` that has the appropriate `backgroundColor` for a given feature, which will give us a bit more flexibility if we want to, say, change the foreground color for such items at the same time.

Reply:
Yeah, I was thinking too that we could add additional colors (like the red and green we use for icons & warnings etc) and then use those references in the styles object. Lots to tidy/consolidate as we move towards more finalized styling!